### DISCUSSION TITLE
[4.0] Adding publication date in related (similar) tags module output?

### DISCUSSION THREAD

**Discussion by jjnxpct** [joomla/joomla-cms#25819] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
I want to show articles in the related tags module but I can't add the publication date to show in my template override.

### Describe the solution you'd like
Add the publication dat (publish_up) - and maybe also the creation an modification date - to the helper.php so we can use in like this: $item->core_publish_up.

I think adding this to the helper file will be enough?
$db->quoteName('cc.core_publish_up'),

And it would also be nice to able to sort the items on this date...

**Comment by ghost** [comment]
New Features will go in J4.

**Comment by jjnxpct** [comment]
OK, thanks. In the mean time I will try to create a custom module based on the similar tags module to see if I can get what I need.

**Comment by brianteeman** [comment]
> I think adding this to the helper file will be enough?
$db->quoteName('cc.core_publish_up'),

Yes that is correct. Just not sure if it should go into core as whee do you stop adding stuff. Should author and category also be added. What about intro image etc

**Comment by jjnxpct** [comment]
> as whee do you stop adding stuff

Good point. I do think pub date is more 'essential' the others. But sure, why not add author and category? ;-) Also, it would be very nice to be able to sort on the pub date... But it's just what I would use / need. So if this is not for the core, no problem for me.
